README.md

```diff
@@ -63,39 +63,12 @@ Do either of the following:
 For iOS development you have to use static libraries. Get the static libraries from the [vosk-api][vosk-api-ios] team.
 
 #### macOS-only
-The Rust linker is not able to extract the correct architecture from a static fat file itself. Therefore, for each architecture we need a non-fat file to link with our code.
-
-Example for aarch64:
-```shell
-lipo -info libvosk.a
-# Architectures in the fat file: libvosk.a are: armv7 armv7s arm64 
-lipo libvosk.a -thin arm64 -output arm64/libvosk.a
-lipo -info arm64/libvosk.a
-# Non-fat file: arm64/libvosk.a is architecture: arm64
-```
-A typical use case is to create a lib using Rust to integrate with an iOS App. So the next step is to add `staticlib` as crate-type in the `Cargo.toml` file.
-The static Vosk-API libraries must be discoverable and linked by the Rust linker. One very feasible approach is to create a [build script][build-script-explanation].
 
-Example build script (build.rs file) for cross compiling aarch64 Android and iOS:
-```rust
-fn add_lib(location: impl AsRef<str>, name: impl AsRef<str>, static_link: bool) {
-    println!("cargo:rustc-link-search={}", location.as_ref());
-    println!(
-        "cargo:rustc-link-lib={}{}",
-        if static_link { "static=" } else { "" },
-        name.as_ref()
-    )
-}
+- [Extract](https://llvm.org/docs/CommandGuide/llvm-lipo.html) the correct non-fat file from the static fat file (libvosk.a) for each architecture you would like to support.
```

Comment:
Is "non-fat" equivalent to "thin"? If so, "thin" would be more concise.

Reply:
Yes, it is the same - I think it depends on the source how it is named. lipo e.g. prints "non-fat file" when you call -info. But the cli parameter -thin extracts the arch-specific file from the "universal" or "fat" binary. Maybe we should name both to avoid confusion?

Reply:
Yeah, that would be perfect